packages/core/package.json

```diff
@@ -424,7 +424,7 @@
         "compileOnly": "tsc -p ./",
         "compileDev": "npm run compile -- --mode development",
         "webpackDev": "webpack --mode development",
-        "serveVue": "webpack serve --config-name vue --mode development",
+        "serveVue": "ts-node ./scripts/build/checkServerPort.ts && webpack serve --port 8080 --config-name vue --mode development",
```

Comment:
is the explicitness just to make it easy to identify that the webpack port and the port in checkServerPort are easy?

Reply:
the explicit `--port 8080` is a fallback in case something breaks in the script, but isn't explicitly needed. Also by default `webpack serve` will use 8080, so I thought it would just help make that more obvious.